Dockerfile

```
@@ -2,7 +2,7 @@ ARG GIT_COMMIT=unspecified
 ARG GIT_REMOTE=unspecified
 ARG VERSION=unspecified
 
-FROM python:3.9.8-alpine
+FROM python:3-alpine
```

Isn't this what we really want to do?
```suggestion
FROM python:3.10.2-alpine
```

I'd suggest creating a new PR with that change and letting Snyk close this one.  I don't know why it wants to throw away version information, but doing so is not in keeping with repeatable builds.

---

You're right @jsf9k - I was thinking we didn't care in this case, but it does indeed go against our repeatability mantra.

---

@jsf9k Yes this should update to either the latest version of Python 3.9 (`3.9.10`) or we can jump to the latest release and use `3.10.2` as your suggestion does. We should also start looking at pinning Alpine versions(`3.9.10-alpine3.14` vs `3.9.10-alpine3.15` for example), but I don't want to over complicate this discussion.